src/sentry/api/urls.py

```diff
@@ -612,6 +613,11 @@
         OrganizationSentryAppInstallationsEndpoint.as_view(),
         name='sentry-api-0-organization-sentry-app-installations'
     ),
+    url(
+        r'^organizations/(?P<organization_slug>[^\/]+)/sentry-app-installations/(?P<uuid>[^/]+)/$',
```

Comment:
I think the last part needs to be `(?P<uuid>[^\/]+)` (escaping the forward slash)? I'm actually not sure what the regex string rules in Python are.